lib/server/middleware/wilson-locals.js

```diff
@@ -17,6 +17,21 @@ module.exports = function(wilsonConfig, hidash) {
 
     res.locals.wilsonConfig = config;
 
+    // Post filtering on config based on wilson-config properties
```

Comment:
If we only need the locales to be filtered in config route it might make more sense to do the filtering there.